content/en/network_monitoring/cloud_network_monitoring/network_analytics.md

```diff
@@ -122,6 +122,30 @@ Using the search bar and the group by feature together is helpful to further iso
 
 {{< img src="network_performance_monitoring/network_analytics/search_bar_with_groupby_2.png" alt="Using group by option with search field" style="width:90%;">}}
 
+### Neutral tags
+
+Neutral tags are tags that are not specific to a client or server but apply to an entire flow. You can search for and filter on traffic with these neutral tags, for example, you can use these tags to filter for traffic that is TLS encrypted.
+
+{{< img src="network_performance_monitoring/network_analytics/cnm_using_neutral_tags.png" alt="Screenshot showing how to search for neutral tags, with an example on searching for tls_encrypted traffic" style="width:90%;">}}
+
+The following is the list of neutral tags available to be used:
+| Tag                     | Description                                                                                   |
+|-------------------------|-----------------------------------------------------------------------------------------------|
+| is_agent_traffic        | Indicates whether the traffic was generated by the Datadog Agent.                             |
+| tls_encrypted           | Specifies if the connection is encrypted using TLS.                                           |
+| tls_cipher_suite        | Identifies the specific TLS cipher suite used (example `TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256`).  |
```

Comment:
(my bad, I know I provided this) can you lowercase the cipher suite example here? we only support querying with lowercase. This is also reflected in the screenshot